setup.py

```diff
@@ -71,10 +71,12 @@
         'Topic :: Software Development :: Libraries :: Python Modules',
         'Topic :: Text Processing :: Markup :: HTML'
     ],
-    packages=['jinja2', 'jinja2.testsuite', 'jinja2.testsuite.res'],
-    install_requires=['MarkupSafe'],
+    packages=['jinja2'],
+    install_requires=[
```

Comment:
This makes py.test a dependency when only installing Jinja2 (even in production). Just remove it, it should be fine.